flang/include/flang/ISO_Fortran_binding.h

```diff
@@ -146,7 +146,16 @@ extern "C++" template <typename T> struct FlexibleArray : T {
   CFI_rank_t rank; /* [0 .. CFI_MAX_RANK] */ \
   CFI_type_t type; \
   CFI_attribute_t attribute; \
-  unsigned char f18Addendum;
+  /* This encodes both the presence of the f18Addendum and the index of the \
```

Comment:
Can these definitions that deal with the `extra` field be moved from this user API header file down into the runtime's `descriptor.h` implementation header?  Application programs don't need to see them.

Reply:
Yes they can. Just moved them.